flang/lib/Optimizer/Transforms/OMPMapInfoFinalization.cpp

```diff
@@ -51,21 +55,66 @@ class OMPMapInfoFinalizationPass
     : public fir::impl::OMPMapInfoFinalizationPassBase<
           OMPMapInfoFinalizationPass> {
 
-  void genDescriptorMemberMaps(mlir::omp::MapInfoOp op,
-                               fir::FirOpBuilder &builder,
-                               mlir::Operation *target) {
-    mlir::Location loc = op.getLoc();
-    mlir::Value descriptor = op.getVarPtr();
+  /// Small helper class tracking a members parent and its
```

Comment:
nit: no need to quantify the size of the helper class 😛.
```suggestion
  /// Tracks a member's parent and its
```